lib/provisioning/account.js

```diff
@@ -47,118 +57,118 @@ function createSubAccount(name, cloudName, customAttributes, enabled, baseAccoun
 }
 
 /**
- * @desc - Delete a subAccount by subAccountID
- * @param subAccountId {string} - The ID generated when using createSubAccount
+ * @desc - Delete the specified sub-account. Supported only for accounts with fewer than 1000 images.
+ * @param sub_account_id {string} - The ID of the sub-account to delete.
  * @param [options] {object} - Generic advanced options map, see online documentation.
  * @param [callback] {function}
  */
-function deleteSubAccount(subAccountId, options = {}, callback) {
-  let uri = ['sub_accounts', subAccountId];
+function delete_sub_account(sub_account_id, options = {}, callback) {
+  let uri = ['sub_accounts', sub_account_id];
   return call_account_api('DELETE', uri, {}, callback, options);
 }
 
 /**
- * @desc - Update a subAccount by subAccountID
- * @param subAccountId {string} - The ID generated when using createSubAccount
- * @param [name] {string} - The name displayed in the management console.
- * @param [cloudName] {string} - Unique identifier, used in API calls
- * @param [customAttributes] {object}
- * @param [enabled] {boolean} - Set the sub-account as enabled or not.
+ * @desc - Update a sub_account by sub_account_id
```

Comment:
What remove the "by sub_account_id" part

Reply:
sub_account_id is the identifier by which you perform the action.
Do you have an idea of how to rename it to make it clearer?

(in theory, this is the same syntax as getElementById, getElementByClassName)